src/owl/config/configure.ml

```diff
@@ -82,7 +83,7 @@ let get_gcc_path _c =
 
 
 let get_openblas_path _c =
-  let p0 = "/usr/local/opt/openblas/lib" in
+  let p0 = "/usr/local/lib/openblas/lib" in
   let p1 = "/opt/OpenBLAS/lib/" in
```

Comment:
configure.ml require extra attention whenever you change it, because something works on your system does not necessarily mean it will work on others. your previous change definitely will break it on some system, e.g. mine. however, you can extend it by adding more rules to detect.